openverse_catalog/dags/providers/provider_workflows.py

```diff
@@ -122,11 +126,11 @@ def __post_init__(self):
         provider_script="phylopic",
         schedule_string="@weekly",
         dated=True,
-        execution_timeout=timedelta(hours=12),
+        pull_timeout=timedelta(hours=12),
     ),
     ProviderWorkflow(
         provider_script="rawpixel",
-        execution_timeout=timedelta(hours=12),
+        pull_timeout=timedelta(hours=12),
     ),
     ProviderWorkflow(
         provider_script="science_museum",
```

Comment:
Do you mind also bumping up Smithsonian's `load_timeout` to 4 hours in this PR?

Reply:
Knew I forgot something! Yes, of course :)